data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -71,68 +125,49 @@ function queue() {
+
+queue resources/apiservices.json oc --request-timeout=5s get apiservices -o json
+queue resources/clusteroperators.json oc --request-timeout=5s get clusteroperators -o json
+queue resources/clusterversion.json oc --request-timeout=5s get clusterversion -o json
+queue resources/configmaps.json oc --request-timeout=5s get configmaps --all-namespaces -o json
+queue resources/csr.json oc --request-timeout=5s get csr -o json
+queue resources/endpoints.json oc --request-timeout=5s get endpoints --all-namespaces -o json
+queue resources/events.json oc --request-timeout=5s get events --all-namespaces -o json
+queue resources/kubeapiserver.json oc --request-timeout=5s get kubeapiserver -o json
+queue resources/kubecontrollermanager.json oc --request-timeout=5s get kubecontrollermanager -o json
+queue resources/machineconfigpools.json oc --request-timeout=5s get machineconfigpools -o json
+queue resources/machineconfigs.json oc --request-timeout=5s get machineconfigs -o json
+queue resources/namespaces.json oc --request-timeout=5s get namespaces -o json
+queue resources/nodes.json oc --request-timeout=5s get nodes -o json
+queue resources/openshiftapiserver.json oc --request-timeout=5s get openshiftapiserver -o json
+queue resources/pods.json oc --request-timeout=5s get pods --all-namespaces -o json
+queue resources/rolebindings.json oc --request-timeout=5s get rolebindings --all-namespaces -o json
+queue resources/roles.json oc --request-timeout=5s get roles --all-namespaces -o json
 # this just lists names and number of keys
-queue resources/secrets-names.txt oc --kubeconfig=/opt/openshift/auth/kubeconfig --request-timeout=5s get secrets --all-namespaces
+queue resources/secrets-names.txt oc --request-timeout=5s get secrets --all-namespaces
 # this adds annotations, but strips out the SA tokens and dockercfg secrets which are noisy and may contain secrets in the annotations
-queue resources/secrets-names-with-annotations.txt oc --kubeconfig=/opt/openshift/auth/kubeconfig --request-timeout=5s get secrets --all-namespaces -o=custom-columns=NAMESPACE:.metadata.namespace,NAME:.metadata.name,TYPE:.type,ANNOTATIONS:.metadata.annotations | grep -v -- '-token-' | grep -v -- '-dockercfg-'
-queue resources/services.json oc --kubeconfig=/opt/openshift/auth/kubeconfig --request-timeout=5s get services --all-namespaces -o json
+queue resources/secrets-names-with-annotations.txt oc --request-timeout=5s get secrets --all-namespaces -o=custom-columns=NAMESPACE:.metadata.namespace,NAME:.metadata.name,TYPE:.type,ANNOTATIONS:.metadata.annotations | grep -v -- '-token-' | grep -v -- '-dockercfg-'
+queue resources/services.json oc --request-timeout=5s get services --all-namespaces -o json
 
-FILTER=gzip queue resources/openapi.json.gz oc --kubeconfig=/opt/openshift/auth/kubeconfig --request-timeout=5s get --raw /openapi/v2
+FILTER=gzip queue resources/openapi.json.gz oc --request-timeout=5s get --raw /openapi/v2
 
 echo "Waiting for logs ..."
 wait
```

Comment:
"Gather remote logs" needs to be here. We can rework the logic
```
export MASTERS=()
if [[ -f ${BOOTSTRAP_IN_PLACE_BOOTSTRAP_PHASE_ARCHIVE_PATH} ]]; then
    # Instead of running installer-masters-gather.sh on remote masters, run it on ourselves
    MASTER_GATHER_ID="master-${GATHER_ID}"
    MASTER_ARTIFACTS="/tmp/artifacts-${MASTER_GATHER_ID}"
    mkdir -p "${ARTIFACTS}/control-plane/master"
    sudo /usr/local/bin/installer-masters-gather.sh --id "${MASTER_GATHER_ID}" </dev/null
    cp -r "$MASTER_ARTIFACTS"/* "${ARTIFACTS}/control-plane/master/"
elif [ "$#" -ne 0 ]; then
    MASTERS=( "$@" )
elif test -s "${ARTIFACTS}/resources/masters.list"; then
    mapfile -t MASTERS < "${ARTIFACTS}/resources/masters.list"
else
    echo "No masters found!"
fi
```

The rest should work 'as-is' as `MASTERS=()`